pkg/logs/input/file/file_provider.go

```diff
@@ -150,8 +152,9 @@ func (p *Provider) searchFiles(pattern string, source *config.LogSource) ([]*Fil
 	sort.SliceStable(paths, func(i, j int) bool {
 		return filepath.Base(paths[i]) > filepath.Base(paths[j])
 	})
+	isWildcardPath := p.containsWildcard(pattern)
 	for _, path := range paths {
-		files = append(files, NewFile(path, source))
+		files = append(files, NewFile(path, source, isWildcardPath))
```

Comment:
I feel like you can just put `true` as we can only enter this method if the the path contains wildcard, see https://github.com/DataDog/datadog-agent/blob/c2636411379c03466615170f217b84dd87b82232/pkg/logs/input/file/file_provider.go#L123

Reply:
Applying the change 👍